Formula/csound.rb

```diff
@@ -29,6 +31,11 @@ class Csound < Formula
   depends_on "portmidi"
   depends_on "stk"
   depends_on "wiiuse"
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
+  uses_from_macos "curl"
+  uses_from_macos "python@2"
```

Comment:
Can it work with python 3?

Reply:
It depends on what is meant by “it”. Csound’s Python bindings (for using Csound from Python) work with Python 3 and are tested with that:

https://github.com/Homebrew/homebrew-core/blob/b6616646f3400ae215b0356f96298f57cea24b42/Formula/csound.rb#L142

Csound’s Python opcodes (for using Python from Csound) don’t work with Python 3 as far as I know. When building Python opcodes on macOS, Csound links against /usr/lib/libpython2.7.dylib. I believe building Python opcodes can be disabled by passing `-DBUILD_PYTHON_OPCODES=OFF` to CMake.

Reply:
👍

Reply:
Thanks @nwhetsell for the explanations!